module/icp/algs/skein/THIRDPARTYLICENSE

```diff
@@ -0,0 +1,3 @@
+Implementation of the Skein hash function.
+Source code author: Doug Whiting, 2008.
+This algorithm and source code is released to the public domain.
```

Comment:
When I ported the ICP originally, there was some talk about not being able to use public domain code because it isn't recognized in some places (like France?). If it's in Illumos though its probably fine.